click-to-reveal/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Click to reveal</title>
+    <link rel="stylesheet" href="./style.css">
+    <script src="./script.js"></script>
+</head>
+
+<body>
+
+    <section id="idt-click-to-reveal">
+        <h1>Click to reveal</h1>
+        <div class="idt-flexBox">
```

Comment:
I don't think the `idt-flexBox` class is really needed. Since we have the `idt-click-to-reveal` wrapper, we can target this div with CSS.

```css
#idt-click-to-reveal div:first-child {
  display: flex;
}
```

Reply:
added this change to the code